tests/mockServer/mock.js

```diff
@@ -880,7 +880,9 @@ function getRun(req, res) {
 
 function getAlerts(req, res) {
   // TODO:ML-8514 Update getAlerts to support both parameters and query strings.
-  res.send({ alerts: alerts.activations })
+  const [paginatedRuns, pagination] = getPaginationConfig(alerts.activations, req.query)
+
+  res.send({ activations: paginatedRuns, pagination })
```

Comment:
It's not runs. Use this code instead
```suggestion
  const collectedAlerts = alerts.activations
  // TODO:ML-8514 Update getAlerts to support both parameters and query strings.

  const [paginatedAlerts, pagination] = getPaginationConfig(collectedAlerts, req.query)

  res.send({ alerts: paginatedAlerts, pagination })
```